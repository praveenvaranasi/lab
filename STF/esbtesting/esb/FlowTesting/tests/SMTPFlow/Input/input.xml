<ns1:Email xmlns:ns1="http://www.fiorano.com/fesb/activity/SMTP1/smtp/in">
   <To>ayrton@fiorano.com</To>
   <From>ayrton@fiorano.com</From>
   <Subject>SMTP Flow Test Case</Subject>
   <Headers>
      <Header name="name" value="value"/>
   </Headers>
   <Attachments>
      <Attachment name="name" readFromFile="no" base64Encoded="no">string</Attachment>
   </Attachments>
   <Body>
      <TextBody>TextBody</TextBody>
      <HtmlBody>HtmlBody</HtmlBody>
   </Body>
</ns1:Email>
