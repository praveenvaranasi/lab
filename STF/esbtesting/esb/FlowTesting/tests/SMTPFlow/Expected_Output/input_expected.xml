<?xml version="1.0" encoding="UTF-8"?>
<ns1:SMTP xmlns:ns1="http://www.fiorano.com/fesb/activity/SMTP1/smtp/out">
   <ns1:Result>Email sent successfully </ns1:Result>
</ns1:SMTP>
